test/tools/libtesteth/ImportTest.cpp

```diff
@@ -241,10 +263,62 @@ bytes ImportTest::executeTest(bool _isFilling)
             if(opt.trValueIndex != -1 && opt.trValueIndex != tr.valInd)
                 continue;
 
-            std::tie(tr.postState, tr.output, tr.changeLog) =
-                    executeTransaction(net, *m_envInfo, m_statePre, tr.transaction);
-            tr.netId = net;
-            transactionResults.push_back(tr);
+            // Skip transaction if there is no expect section for this transaction (noindex data,
+            // gas, val, network)
+            bool found = false;
+            json_spirit::mArray expects;
+            if (_isFilling)
+                expects = m_testInputObject.at("expect").get_array();
+            else
+                expects = m_testInputObject.at("post").get_obj().at(netIdToString(net)).get_array();
+
+            for (auto const& exp : expects)
+            {
+                json_spirit::mObject const& expSection = exp.get_obj();
+
+                if (_isFilling)
+                {
+                    set<string> networkSet;
+                    parseJsonStrValueIntoSet(expSection.at("network"), networkSet);
```

Comment:
Can this and the next line be done outside of for loops?

Reply:
no this is done for each expect section

Reply:
but you right this information could be gathered once. before the loops. it's just the plain way to do this. more logical objects to store the information about expect section required. I don't want to change the code very much. I hope it is the last change. and this code will soon become obsolite / replaced by retesteth